### DISCUSSION TITLE
Getting "GET /versions.json HTTP/1.1" code 404 warnings since removing Mike

### DISCUSSION THREAD

**Discussion by Andre601** [jimporter/mike#170] [Q&A] [answered]
I used Mike in the past but due to multiple reasons did I decide to remove it.

The issue I encounter right now is, that MkDocs constantly reports `"GET /versions.json HTTP/1.1" code 404` in the console when using `mkdocs serve` and the page being rebuild.

I have no idea why MkDocs tries to find this file. I have it not defined within the nav...
What could possible reasons be and how would I fix them?

**Comment by jimporter** [comment]
Unfortunately, this isn't enough information for me to diagnose the problem. Please [file a bug](https://github.com/jimporter/mike/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=) for this and fill out as many of the details in the form as possible.

**Reply by Andre601** [reply, marked as answer]
Figured it out... I accidentally left a config option in the mkdocs.yml file causing this issue...
